README.md

```diff
@@ -25,7 +25,8 @@ There are multiple versions/ tags available under [dockerhub:unihalle/maximapool
 
 | Tag                | STACK version | OS/Tomcat/JRE   | Maxima version | assStackQuestion | ILIAS | moodle-qtype_stack |
 |:------------------ | -------------:| --------------- | --------------:| ----------------:| ----- | ------------------ |
-| `latest`           | 2017121800    | debian sid/9/10 | 5.41.0-Linux   | 12439ff          | 5.3   |4.1
+| `latest`           | 2018030500    | debian sid/9/10 | 5.41.0-Linux   |                  |       |4.1+ [(9bf7a7f)](https://github.com/maths/moodle-qtype_stack/tree/9bf7a7ff6118086480f2b3db5fbb933150c8fa49)
```

Comment:
Looks like the submodule moodle-qtype_stack is on [d5c8cd6](https://github.com/maths/moodle-qtype_stack/commit/d5c8cd6) (_Update Maxima version numbers for the 4.2.1 release._, Aug 6, 2018) not on [9bf7a7f](https://github.com/maths/moodle-qtype_stack/commit/9bf7a7f) (_Update stackmaximaversion to correctly identify when the file was last changed_,  Apr 19, 2018)

Reply:
Thanks, I obviously messed that up somehow! I've pushed an update that corrects this.